cfme/utils/__init__.py

```diff
@@ -191,13 +190,13 @@ def safe_string(o):
     Args:
         o: Anything.
     """
-    if not isinstance(o, six.string_types):
+    if not isinstance(o, str):
```

Comment:
I'm suspect; I think, now we won't need this.

Reply:
safe_string the whole method? Agreed - I toyed with this though, but decided to just convert for now and remove in separate PR, as it was a rabbit hole separate from using `six`.